config/templates.js

```diff
@@ -1,32 +1,40 @@
 module.exports.templates = {
```

Comment:
We'll need to reorder these so that the ones we recommend (uswds, team, landing-page) are on the top row.